include/itkCuberilleImageToMeshFilter.hxx

```diff
@@ -29,16 +29,17 @@ namespace itk
 
 template<class TInputImage, class TOutputMesh, class TInterpolator>
 CuberilleImageToMeshFilter<TInputImage,TOutputMesh,TInterpolator>
-::CuberilleImageToMeshFilter()
+::CuberilleImageToMeshFilter() :
+  m_IsoSurfaceValue( NumericTraits< InputPixelType >::One ),
+  m_MaxSpacing( NumericTraits< SpacingValueType >::One ),
```

Comment:
@thewtex @lorensen would any of you happen to have some time to review this?
The only controversial point may be this one.
I'll hopefully merge the PR once you review it. Thanks.